.github/workflows/release.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
           CIRCUITS_ROOT: ${{ runner.temp }}/zeto-artifacts
           PROVING_KEYS_ROOT: ${{ runner.temp }}/zeto-artifacts
           PTAU_DOWNLOAD_PATH: ${{ runner.temp }}/zeto-artifacts
-        working-directory: zkp/circuits
+        working-directory: zkp/circuits/scripts
```

Comment:
this should be reverted back to the zkp/circuits folder, to be consistent with how `npm run gen` references the program (as `scripts/gen.js`)